spec/element_base_spec.rb

```diff
@@ -0,0 +1,94 @@
+require 'spec_helper'
+
+RSpec.describe Kitchen::ElementBase do
+
+  let(:element) do
```

Comment:
it is true that this returns an element (a `BookElement`).   Since everything is an element (including `para` below), might be more clear to call this `:book`